Makefile

```diff
@@ -184,4 +184,8 @@ git-rebase:
 clean-cluster:
 	(cd $(TEST_DIR) && ./clean-cluster.sh)
 
-.PHONY: all fmt container clean release semver-cli
+# list the available makefile targets
+list:
```

Comment:
As i tried this on my machine, it doesn't seem to fully list all the targets. The below line did: 

```
sh -c "$(MAKE) -p no_targets__ | awk -F':' '/^[a-zA-Z0-9][^\$$#\/\\t=]*:([^=]|$$)/ {split(\$$1,A,/ /);for(i in A)print A[i]}' | grep -v '__\$$' | sort"`
```